packages/coding-standard/src/Fixer/AbstractSymplifyFixer.php

```diff
@@ -41,23 +40,6 @@ protected function reverseTokens(Tokens $tokens): array
         return array_reverse($tokens->toArray(), true);
     }
 
-    /**
-     * Helper method to run this before specified fixer, works even in case of change.
-     */
-    protected function getPriorityBefore(string $fixerClass): int
-    {
-        if (! is_a($fixerClass, FixerInterface::class, true)) {
-            return 0;
-        }
-
-        $reflectionClass = (new ReflectionClass($fixerClass));
-
-        /** @var FixerInterface $fixer */
-        $fixer = $reflectionClass->newInstanceWithoutConstructor();
-
-        return $fixer->getPriority() + 5;
-    }
```

Comment:
Fixes a bug then `getPriority()` was called on a `ProxyFixer` that required `__construct()` to be called.

![image](https://user-images.githubusercontent.com/924196/117014003-abd62a00-acf0-11eb-89d8-fb5c1856fab5.png)